_playwright-tests/tests/API/PopularRepos.spec.ts

```diff
@@ -9,4 +9,47 @@ test.describe('Popular repositories', () => {
     expect(resp.meta?.count).toBe(1);
     expect(resp.data?.[0].suggestedName).toBe('EPEL 9 Everything x86_64');
   });
+
+  test('test_popular_repos_pagination_api', async ({ client }) => {
+    await test.step('Test limit parameter - get first repository only', async () => {
+      const firstRepo = await new PopularRepositoriesApi(client).listPopularRepositories({
+        limit: 1,
+      });
+
+      expect(firstRepo.data?.length).toBe(1);
+      expect(firstRepo.data?.[0].suggestedName).toBe('EPEL 10 Everything x86_64');
+    });
+
+    await test.step('Test offset parameter - skip first repository', async () => {
+      const secondRepo = await new PopularRepositoriesApi(client).listPopularRepositories({
+        offset: 1,
+      });
+
+      expect(secondRepo.data?.length).toBe(2); // Should get repos 2 and 3
+      expect(secondRepo.meta?.count).toBe(3); // Total count should still be 3
```

Comment:
the count is verified in this step and the last one, both of each only use the offset query param, might be nice to add it to the other steps, the ones using the limit param, to verify that the count is still correct while using that param 👍🏼🔧